sdk-internals/communicator/ssh/communicator_test.go

```diff
@@ -230,3 +230,34 @@ func TestHandshakeTimeout(t *testing.T) {
 		t.Fatalf("Expected handshake timeout, got: %s", err)
 	}
 }
+
+func TestIsExclude(t *testing.T) {
+	testCases := []struct {
+		Exclude []string
+		Path    string
+		Is      bool
```

Comment:
In terms of clarity, this bool seems to be used to check that we did exclude something, could you rename it to make it clearer what it's used for?
That's personal preference, but I found that constructions like `ExpectExclude` works well at communicating the intent, but feel free to give it your own take on that!

Reply:
Nice suggestion, applied!